openmls/Cargo.toml

```diff
@@ -46,15 +46,16 @@ crypto-debug = [] # ☣️ Enable logging of sensitive cryptographic information
 content-debug = [] # ☣️ Enable logging of sensitive message content
 
 [dev-dependencies]
+backtrace = "0.3"
 criterion = "^0.4"
-pretty_env_logger = "0.4"
+hex = { version = "0.4", features = ["serde"] }
```

Comment:
This is a new dev-dependency. It allows to use ...

```rust
pub struct MessagesTestVector {
    #[serde(with = "hex::serde")]
    mls_welcome: Vec<u8>,
    // ...
```

... to get rid of hex-to-bytes conversions.